EpsilonDriverControls/Src/DriverControls.c

```diff
@@ -100,6 +102,21 @@ void sendDriverTask(void const* arg)
         hcan2.pTxMsg->Data[1] = 0;
         hcan2.pTxMsg->Data[2] = 0;
         hcan2.pTxMsg->Data[3] = 0;
+
+        if (HAL_ADC_PollForConversion(&hadc1, 1000000) == HAL_OK)
+        {
+            regen_ADCValue = HAL_ADC_GetValue(&hadc1);
+        }
+        hcan2.pTxMsg->Data[0] += (regen_ADCValue & 0x000000ffUL);
```

Comment:
I think `=` would be better here and below use `|=` instead of `+=`. They most likely could be the same but `|=` is more explicit of what you are doing (and if it is somehow signed there can be no overflow).